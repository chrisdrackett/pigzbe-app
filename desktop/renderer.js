import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
