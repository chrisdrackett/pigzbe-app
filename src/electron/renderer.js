import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
