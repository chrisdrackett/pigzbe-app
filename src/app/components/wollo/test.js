import React from 'react';
import renderer from 'react-test-renderer';
import Wollo from './';

const props = {
    balance: '100.0000'
};

describe('Wollo', () => {
    test('renders correctly', () => {
        const tree = renderer.create(<Wollo {...props}/>).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
