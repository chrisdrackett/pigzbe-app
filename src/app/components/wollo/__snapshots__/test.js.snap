// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wollo renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/app/components/wollo/images/currency_logo.png",
        }
      }
      style={
        Object {
          "height": 19,
          "marginRight": 10,
          "width": 23,
        }
      }
    />
    <Text
      style={
        Object {
          "color": "rgb(255, 255, 255)",
          "fontFamily": "Poppins",
          "fontSize": 22,
          "fontWeight": "bold",
        }
      }
    >
      100
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "rgba(255, 255, 255, 0.6)",
        "fontFamily": "Poppins",
        "fontSize": 14,
        "fontWeight": "bold",
      }
    }
  >
    Wollo balance
  </Text>
</View>
`;
